assets/javascripts/modules/checkout/deliveryDetails.js

```diff
@@ -134,8 +134,7 @@ define([
             }
 
             bean.on(formEls.$DELIVERY_DETAILS_SUBMIT[0], 'click', handleBillingVisibility);
-
-            if (formEls.DELIVERY.$POSTCODE_CONTAINER.data('validate-for') === 'delivery') {
+            if (formEls.DELIVERY.$POSTCODE_CONTAINER.data('validate-for') === 'delivery.address') {
```

Comment:
Is this the correct way to fix this ?
i see that a similar change was done recently for [vouchers](https://github.com/guardian/subscriptions-frontend/pull/1224/files)
After these changes the home delivery validation works for there is no voucher validation at all. I know the postcode whitelist should not be applied but it doesn't even check that the string entered is a real postcode.
is this the expected behaviour @paulbrown1982 ?

Reply:
Yeah this is the correct way to fix. My GW Gifting work nested the name prefix one level deeper for delivery address, and I missed updating this selection.